pkg/apis/akash.network/crd.yaml

```diff
@@ -183,3 +188,62 @@ spec:
                   type: integer
                 oseq:
                   type: integer
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  # name must match the spec fields below, and be in the form: <plural>.<group>
+  name: providerleasedips.akash.network
+spec:
+  # group name to use for REST API: /apis/<group>/<version>
+  group: akash.network
+  scope: Namespaced
+  # list of versions supported by this CustomResourceDefinition
+  versions:
+    - name: v2beta1
+      # Each version can be enabled/disabled by Served flag.
+      served: true
+      # One and only one version must be marked as the storage version.
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                lease_id:
+                  type: object
+                  properties:
+                    owner:
+                      type: string
+                    dseq:
+                      type: string
+                      format: uint64
+                    gseq:
+                      type: integer
+                    oseq:
+                      type: integer
+                    provider:
+                      type: string
+                service_name:
+                  type: string
+                port:
+                  type: integer
+                external_port:
+                  type: integer
+                protocol:
+                  type: string
+                sharing_key:
+                  type: string
```

Comment:
OT: it'd be great if we could re-use schemas - have `LeaseID` defined in one place for instance.  Not sure if it's possible.